game.c

```diff
@@ -172,7 +172,7 @@ void set_options(long int *map_seed, long int *events_seed, bool *reveal_map) {
 
 int main() {
     int ch;
-    int turn = 0;
+    int turn = 1;
```

Comment:
The simulation assumes that the clock starts at 0. Either you need to change that or keep that constraint here. Otherwise you'll see weird behavior in the first turn.

Reply:
Not a problem. I'll just print 'n + 1' which - horrifingly - may be how the n++ got in there. I hope I didn't do that, but....